postgres/README.md

```diff
@@ -0,0 +1,191 @@
+# TDK PostgreSQL Demo
+
+
+[Pagila](https://github.com/devrimgunduz/pagila) is a sample database schema that can be used in MySQL tutorials, articles, demos, etc.
```

Comment:
Is MySQL mentioned here by mistake?

Reply:
Fixed in [384d5ba](https://github.com/synthesized-io/pagila-tdk-demo/pull/12/commits/384d5ba92ac472b127fd1688589afe1251a3425f)